### ISSUE TITLE
Should probably relax REFL003 if the type comes from GetType() and is not sealed.

### ISSUE THREAD

**Issue by brian-reichle** [DotNetAnalyzers/ReflectionAnalyzers#108]
REFL003 probably shouldn't complain about missing members if it doesn't know the exact type (or should at least report them under a different id so the severity can be adjusted), as the reflected member may well be provided by a derived class.

```csharp
ParameterInfo[] GetParameters(Delegate i) => i.GetMethod("Invoke").GetParameters();
```
